recipes/regiis.rb

```diff
@@ -28,4 +28,5 @@
   command "%WINDIR%\\Microsoft.Net\\Framework64\\v4.0.30319\\aspnet_regiis.exe -i -enable > #{guard_file}"
   creates guard_file
```

Comment:
I invite you to read the documentation about the `creates` property:
https://docs.chef.io/resource_execute.html#properties